homeassistant/components/onboarding/views.py

```diff
@@ -430,9 +430,19 @@ async def post(self, manager: BackupManager, request: web.Request) -> web.Respon
         return await self._post(request)
 
 
-def setup_cloud_views(hass: HomeAssistant, data: OnboardingStoreData) -> None:
+async def setup_cloud_views(hass: HomeAssistant, data: OnboardingStoreData) -> None:
     """Set up the cloud views."""
 
+    with async_pause_setup(hass, SetupPhases.WAIT_IMPORT_PACKAGES):
+        # Import the cloud integration in an executor to avoid blocking the
+        # event loop.
+        def import_cloud() -> None:
+            """Import the cloud integration."""
+            # pylint: disable-next=import-outside-toplevel
+            from homeassistant.components.cloud import http_api  # noqa: F401
```

Comment:
dont we need to remove the import on line 450?

Reply:
Nevermind, that won't work due to the scoping in this function. I'd just leave it like this.